topgun/topgun_suite_test.go

```diff
@@ -491,7 +491,7 @@ func waitForWorkerInState(desiredStates ...string) string {
 		}
 
 		return workerName
-	}).ShouldNot(BeEmpty())
+	}).ShouldNot(BeEmpty(), "should have seen a worker in states: "+strings.Join(desiredStates, ", "))
```

Comment:
Whoa..I had no idea this was a thing.

Reply:
yeah, much more helpful than 'expected string not to be empty' lol